opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -304,6 +308,24 @@ namespace Opm
 
 
 
+    template<typename TypeTag>
+    typename StandardWell<TypeTag>::Eval
+    StandardWell<TypeTag>::getPerfPressure(const typename StandardWell<TypeTag>::FluidState& fs) const
```

Comment:
This is not the perforation pressure. It is the pressure in the perforated grid cell.